packages/react/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -84,6 +84,13 @@ export interface IDropdownProps
    */
   dropdownWidth?: number;
 
+  /**
+   * Adjust the width to fit the content.
+   * Mutually exclusive with `dropdownWidth`.
+   * @defaultvalue false
+   */
+  isDropdownWidthFitContent?: boolean;

```

Comment:
Looks good overall. Only suggestion is that I think a name like `growDropdownWidth` might be better. It's not blocking though so I've approved.

Reply:
I wonder if instead of adding a property, it would be better to change the type of `dropdownWidth` to be `dropdownWidth?: number | 'auto'` -- that way we don't have to add another prop.

Reply:
@ecraig12345 I think `auto` is not a good name. Because when `dropdownWidth` equal to 0 or `undefined`, it also has some kind of auto behavior (the width of callout always equal to the width of dropdown box).

what about `dropdownWidth?: number | 'grow'`?

Reply:
@joschect @ecraig12345 I have updated it. Please have a look.

Reply:
I was thinking "auto" since it's also used for some CSS attributes.

Reply:
Agree with @ecraig12345. Unless the semantics of the auto/grow here are significantly different from how 'auto' tends to be behave in CSS, we should follow the convention and use 'auto'.

Reply:
`auto` can have a variety of effects in CSS, but for size-related properties (such as the examples mentioned in [this stackoverflow answer](https://stackoverflow.com/a/4471874/890624)) the semantics are similar to what we want here.

Reply:
@ecraig12345 Updated. Please check.